index.html

```diff
@@ -13,16 +13,34 @@
 
     <script>
         var calculator = {
-            addNumbers: function () {
+            addNumbers: function (numberOne, numberTwo) {
                 // hint: these are strings
                 var numberOne = document.forms.inputs.numberOne.value;
                 var numberTwo = document.forms.inputs.numberTwo.value;
-                document.forms.inputs.result.value = numberOne + numberTwo;
+                document.forms.inputs.result.value = parseInt(numberOne) + parseInt(numberTwo);
             },
-            subtractNumbers: function () {
+            subtractNumbers: function (numberOne, numberTwo) 
+            { var numberOne = document.forms.inputs.numberOne.value;
+                var numberTwo = document.forms.inputs.numberTwo.value;
+                document.forms.inputs.result.value = parseInt(numberOne) - parseInt(numberTwo);
+            },
+                mulitplyNumbers: function (numberOne, numberTwo)
+             {var numberOne = document.forms.inputs.numberOne.value;
+                var numberTwo = document.forms.inputs.numberTwo.value;
+                document.forms.inputs.result.value = parseInt(numberOne) * parseInt(numberTwo);
+            },
+                divideNumbers: function (numberOne, numberTwo) {
+                    var numberOne = document.forms.inputs.numberOne.value;
+                var numberTwo = document.forms.inputs.numberTwo.value;
+                document.forms.inputs.result.value = parseInt(numberOne) / parseInt(numberTwo);
+            },
+
```

Comment:
The code above all looks correct.  The formatting is not required for code to work. It is important to learn for readability by you and others.